docs/reference/triggers/v3iostream.md

```diff
@@ -100,56 +100,84 @@ As of Nuclio v1.1.33 / v1.3.20, you can configure the following configuration pa
-```bash
+The easiest way to set up a stream is with the [`v3ctl`](https://github.com/v3io/v3ctl) platform CLI.
+[Download the latest CLI release](https://github.com/v3io/v3ctl/releases), rename the executable binary to **v3ctl**, and add executable permissions by running the following command from a command-line shell:
+```sh
+chmod +x v3ctl
+```
+
+> **Note:**
+> - When running remotely, from outside of the platform, you need to add the `--access-key` and `--webapi-url` options to all `v3ctl` commands to provide a valid access key and the API URL of the web-APIs service for your platform environment.
+> - For full usage instructions, including additional options, run `v3ctl <command> --help`.
+>   For example, `v3ctl create stream --help` or `v3ctl create stream record --help`.
+
+Run the following from a command-line shell to create a platform stream named "test-stream-0" (see the stream-path argument) in the predefined "users" platform data container (`--container`).
+The stream has a retention period of 24 hours (`--retention-period`) and 32 shards (`--shard-count`):
+```sh
 ./v3ctl create stream test-stream-0 \
     --container users \
     --retention-period 24 \
     --shard-count 32
 ```
 
-> Note: If you're running outside of the platform, you'll need to provide --access-key and --webapi-url as well to all v3ctl commands
-
-Now create a few records across all the shards, specifying the shard:
-
-```bash
+Now, use the `create stream record` CLI command to add 10 records (IDs `1`-`10`) to each stream shard (`--shard-id`, which is set to a zero-based shard ID - `0`-`9`).
+For test purposes, the ingested record data in the example is a string denoting the shard and record IDs - `shard-$shard_id-record-$record_id` (`--data`).
+As in the `create stream` command, the stream path is set to a "test-stream-0" stream in the root directory of the "users" data container, by using the stream-path argument and the `--container` option.
+```sh
 for shard_id in {0..31}
 do
     for record_id in {1..10}
     do
-        ./v3ctl create stream record \
+        ./v3ctl create stream record /test-stream-0 \
             --container users \
             --shard-id $shard_id \
-            --data shard-$shard_id-record-$record_id /test-stream-0
+            --data shard-$shard_id-record-$record_id
     done;
 done;
 ```
 
-The stream is ready for consumption by a Nuclio function. Deploy a Python function that logs the body and sleeps for 5 seconds:
+After the command completes successfully, the stream is ready for consumption by a Nuclio function. To test this, do the following from the dashboard's **Projects** page, to define and deploy a function that consumes the stream records and uses a `v3ioStream` trigger.
 
-```python
-import time
+> **Note:** Before you deploy the function, ensure that the log-forwarder service is enabled (see the **Services** dashboard page), so that you can view the function logs.
```

Comment:
You don't have to. You can use kubectl in Jupyter / shell to see the logs (as I have). Log forwarder is for folks that want to ship all their logs to an external ElasticSearch cluster.

Reply:
OK. I removed the note and edited the references to the logs later on (see my reply to your related comment) — commit f61916f.